.agent/workflows/optimize.md

```
@@ -0,0 +1,212 @@
+
+| File | Baseline Tokens | Candidates | Severity | Risk |
+|------|-----------------|------------|----------|------|
+| *target file* | X | N changes | Safe/Moderate/Significant | note |
+
+---
+
+## 3. Optimization Rules
+
+Apply in order.
+
+### 3a. Eliminate _(§Documentation: Anti-patterns)_
```

markdownlint-cli2 (0.21.0)

[warning] Emphasis style
Expected: asterisk; Actual: underscore

(MD049, emphasis-style)